base/deprecated.jl

```diff
@@ -778,6 +778,18 @@ end
 @deprecate cholfact!(A::Base.LinAlg.HermOrSym, uplo::Symbol, ::Type{Val{false}}) cholfact!(A, Val{false})
 @deprecate cholfact!(A::Base.LinAlg.HermOrSym, uplo::Symbol = :U) cholfact!(A)
 
+# #18218
+function arithtype(T)
```

Comment:
I don't think I can because the macro is when a function should be used instead of another one. Here there is not replacement.